src/main/resources/templates/index.html

```diff
@@ -54,12 +54,6 @@
             <span class="nav-link-text">Test Rules</span>
           </a>
         </li>
-        <li id="eiInfoBtn" class="nav-item" data-toggle="tooltip" data-placement="right" title="Eiffel Intelligence Information">
```

Comment:
This is not the button I mentioned in the bug story. I will update the story.

Reply:
Well this button is not showing content.

The button you mentioned is information icon.... a common way for user information. It inform user about the use of AND and OR condition. I think it should be there for user help.  Do you still think that it should be removed...?